docker-compose/docker-compose.yaml

```diff
@@ -18,7 +18,8 @@ services:
       test: ['CMD', "mariadb-admin", "--password=password", "--protocol", "tcp", "ping"]
```

Comment:
@grooverdan  / @fauust the check may not work when we start the container with PROD backup, because:

-> currently on PROD: root has `unix_socket` auth type
-> currently on DEV: root has `mysql_native_password` auth type

So when we switch to containers, **this will not work**:
`"mariadb-admin", "--password=password"`

**My question is:**
-> during container startup, **with Production backup restored**, does the `healthcheck.sh` script gets recreated together with the database permissions so we can use it consistently in both DEV and PROD?

In DEV I already checked that we have the user and the script, working. So I'll update the `test: `command. Thanks @grooverdan

Reply:
healthcheck.sh uses a healthcheck user, so env `MARIADB_AUTO_UPGRADE=1` will create it if not there. So include this too.

Reply:
It is, declared as environment in docker-compose,
- MARIADB_AUTO_UPGRADE=1